warehouse/age-off/src/test/java/datawave/iterators/filter/ConfigurableAgeOffFilterTest.java

```diff
@@ -1,9 +1,6 @@
 package datawave.iterators.filter;
 
-import static org.hamcrest.CoreMatchers.is;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertSame;
+import static org.junit.Assert.*;
```

Comment:
Avoid wildcard imports. 
```suggestion
import static org.junit.Assert.assertFalse;
import static org.junit.Assert.assertNotNull;
import static org.junit.Assert.assertSame;
import static org.junit.Assert.assertTrue;
```

Reply:
IntelliJ being "helpful" there....